gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Oct 29 13:37:47 PDT 2020
+#Sat Jan 28 22:11:42 WAT 2023
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
```

Comment:
you should revert your code changes. the only real change is from getting "all" to "binary" which "all" is better for debugging. There is also some reordering of lines